packages/react/src/components/header/components/headerActionBar/HeaderActionBar.module.scss

```diff
@@ -32,6 +33,13 @@
   width: 100%;
 }
 
+.mobileMenuContainer {
+  height: 100%;
+  overflow-x: hidden;
+  overflow-y: auto;
+  position: fixed;
+}
```

Comment:
There is potential problem caused by this. Height of the whole `<header>` is 0px in the dom. Some elements below the header can be positioned incorrectly. The header should at least have the height of the actionBar.

The fixed element should be a descendant, maybe the navigationMenu, which is already absolutely positioned.

Reply:
Ah, that is true. I set the position: fixed so high in DOM tree because that way the whole actionbar and mobile menu cover the whole page. You can see this in doc site demo and remove the position fixed rule, then the whole page comes into view by scrolling after the menu. But even with the position fixed, the scroll bar appears in doc site (when the page is long), and the parent height: 0, this is not ideal... Sigh. Back to the drawing board.

Reply:
I kept the `position: fixed` because it covers the whole page so well and just set min-height for the Header so it takes up the space of the action bar.